src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -0,0 +1,1108 @@
+namespace System.Security {
+  public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk {
+    protected CodeAccessPermission() { }
+    public void Assert() { }
+    public abstract System.Security.IPermission Copy();
+    public void Demand() { }
+    public void Deny() { }
```

Comment:
We should make sure that members like this one which are [Obsolete] on desktop are also obsolete here, like Deny.